src/main/java/nvt/kts/ticketapp/repository/event/EventDaysRepository.java

```diff
@@ -10,4 +13,10 @@
 @Repository
 public interface EventDaysRepository extends JpaRepository<EventDay, Long> {
     List<EventDay> findAllByDate(Date date);
+
+    @Query("select day from EventDay d where d.event.id = :id")
```

Comment:
Nepotreban query.. query anotaciju koristi samo kad su neki slozeni upiti. Sve to mozes resiti tako sto ces napisati List<EventDay>findAllByEventId(Long eventId); to je sve
pogledaj u dokumentaciji     https://docs.spring.io/spring-data/jpa/docs/current/reference/html/#jpa.query-methods.named-queries  sekciju 5.3.2 kako se prave upiti

Reply:
Obrisano